brainframe/cli/frozen_utils.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from pathlib import Path
+
+
+def is_frozen() -> bool:
+    """
+    :return: True if the application is running as a frozen executable
+    """
+    return getattr(sys, "frozen", False)
+
+
+RELATIVE_TRANSLATIONS_PATH = Path("brainframe/cli/translations")
```

Comment:
Thoughts on doing a pattern similar to what the client does where we'd have all the paths defined like this:

```python
RELATIVE_TRANSLATIONS_PATH = route_path("brainframe/cli/translations")
RELATIVE_DEFAULTS_FILE_PATH = route_path("brainframe/cli/defaults.yaml")
# ... other paths ...

def resource_path(*args: Union[str, Path]):
   if _running_in_build():
        return Path(sys._MEIPASS, *args)
    elif _running_in_snap():
        return Path(find_client_root(), *args)
    return Path(*args)

```

This had extra benefits where we could check at import-time that all the paths necessary for the program to run exist.

It also reduces the duplicate logic of path conversion.

Reply:
I'm down! I don't want paths to be checked at import-time because it makes providing human-readable error messages a lot uglier, but resolving the path at import-time is cool with me. This has the benefit of removing the problem with the `FileNotFoundError`.

Reply:
Did it!

Reply:
Running at import time is probably a good idea though, because it's not about providing human-readable or translated error messages. Instead, it allows you to make sure everything is packaged right off the bat. 

So in CI, you can make a build, then run `brainframe --help`, and if any files weren't packaged that would fail the build. 

I think that it's worth considering. It definitely helped prevent issues on the client in the past. It also makes it easier from your perspective- you can always expect the file to exist!

Reply:
Okay, that's a good point.

Reply:
Did it!